GramAddict/plugins/interact_usernames.py

```diff
@@ -214,7 +214,7 @@ def interact():
                 with open(current_file, "w") as f:
                     f.writelines(remaining)
         else:
-            logger.warning(f"File {current_file} not found.")
+            logger.warning(f"File {current_file}.txt not found.")
```

Comment:
I don’t understand this. We shouldn’t assume a .txt extension and the full filename should be in current file.

Reply:
You removed the part where I added the txt at the end. If you don't write it, it says file non found. So I got myself in error 😊

Reply:
Yeah, we shouldn’t assume it’s a .txt. It could be any extension or even no extension. As long as it’s line by line, that’s all that matters. It’s user error if you forget to add the .txt at the end of the filename. Lol